src/main/java/seedu/address/model/person/Role.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's role in the address book.
+ */
+public class Role {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Roles should only be 'brUdder' or 'mUdder' and it can't be blank.";
+
+    private String role;
+
+    /**
+     * Constructs a {@code Role}.
+     *
+     * @param role a valid role.
+     */
+    public Role(String role) {
+        requireNonNull(role);
+        checkArgument(isValidRole(role), MESSAGE_CONSTRAINTS);
+        this.role = role;
+    }
+
+    /**
+     * Returns true if a given role string is a valid role.
+     */
+    public static boolean isValidRole(String test) {
+        return test.equals("brUdder") || test.equals("mUdder");
```

Comment:
same thing, I think we need to make the validation case insensitive but make it be displayed as 'brUdder' or 'mUdder' only. Otherwise, it'll be an inconvenience for the users.